modules/oxide-components/src/main/ts/contexts/UniverseContext/UniverseProvider.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { UniverseContext } from './UniverseContext';
+import type { UniverseResources } from './UniverseTypes';
+
+interface UniverseContextProvider {
+  readonly resources: UniverseResources;
+  readonly children: React.ReactNode;
+}
+
+export const UniverseProvider = ({
```

Comment:
I'm not a fan of the word `universe` as an API, our other use of it is in an internal library. This will one day become TinyMCE's public API.

But names are hard and I don't want to have that discussion today.